modules/ROOT/pages/setting-up-your-development-environment.adoc

```diff
@@ -39,7 +39,7 @@ Java 7 is compatible with Mule 3.6 and below.
 
 [NOTE]
 --
-https://archive.docs.mulesoft.com/release-notes/studio/anypoint-studio-6.1-with-3.8.1-runtime-update-site-1-release-notes[Studio 6.1.1] and newer allows you to log-in to Anypoint Platform using your configured xref:access-management::external-identity.adoc[federated external identity].
+Studio 6.1.1 and newer allows you to log-in to Anypoint Platform using your configured xref:access-management::external-identity.adoc[federated external identity].
```

Comment:
```suggestion
Studio versions 6.1.1 and later allow you to log in to Anypoint Platform using your configured xref:access-management::external-identity.adoc[federated external identity].
```
Another option would be: "Studio 6.1.1 and later versions..."

As per our guidelines: 
"later"
Use instead of "above" or "greater" to refer to subsequent releases.